tox.ini

```diff
@@ -35,5 +35,4 @@ deps=mypy
 setenv=MYPYPATH={toxinidir}
 # TODO: Drop --ignore-missing-imports once we have type annotations for eth_utils, coincurve and cytoolz
 commands=
-    mypy --py2 --ignore-missing-imports -p eth_keys
```

Comment:
If we're no longer supporting py2, we could start migrating those type annotations to py3-style!

Reply:
We're good to drop `py2` across the board on every python tool that I've written so 👍